usersession/userd/privileged_desktop_launcher.go

```diff
@@ -290,12 +315,14 @@ func parseExecCommand(command string, icon string) ([]string, error) {
 			arg = arg[1:]
 		} else if strings.HasPrefix(arg, "%") {
 			switch arg {
-			case "%f", "%F", "%u", "%U":
-				// If we were launching a file with
-				// the application, these variables
-				// would expand to file names or URIs.
-				// As we're not, they are simply
-				// removed from the argument list.
+			case "%f", "%u":
```

Comment:
I think we want to handle %f and %u differently: if the desktop file uses `%f`, then we want to check that the URI is a file URI, and extract the unescaped path component.